docs: user tutorial for GCP NFS (#332)
kyma-project/cloud-manager: docs/user/tutorials/01-20-gcp-nfs-volume.md

   gcpnfsvolume.cloud-resources.kyma-project.io/my-vol condition met
   ```
   
4. Observe the generated PersistentVolume:
   
   ```shell
   export PV_NAME=`kubectl get gcpnfsvolume my-vol -n $NAMESPACE -o jsonpath='{.status.id}'`
   kubectl get persistentvolume $PV_NAME
   ```
   
   You should see the following details:
   
   ```
   NAME       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                    STORAGECLASS
   {PV_NAME}  1Ti        RWX            Retain           Bound    {NAMESPACE_NAME}/my-vol                             
   ```
   
   Note the `RWX` access mode which allows the volume to be readable and writable from multiple workloads, and the
   the `Bound` status which means the PersistentVolumeClaim claiming this PV is created.
   
5. Observe the generated PersistentVolumeClaim:

   ```shell
   kubectl -n $NAMESPACE get persistentvolumeclaim my-vol
   ```

   You should see the following message:

   ```
   NAME     STATUS   VOLUME     CAPACITY   ACCESS MODES   STORAGECLASS 
   my-vol   Bound    {PV_NAME}  1Ti        RWX                                                
   ```

   Similarly to PV, note the `RWX` access mode and `Bound` status.

> 6. Create two workloads that both write to the volume

Reviewer: ```suggestion
6. Create two workloads that both write to the volume. Run:
```

       metadata:
         labels:
           app: gcpnfsvolume-demo
       spec:
         containers:
           - name: my-container
             image: ubuntu  
             command: 
               - "/bin/bash"
             args:
               - "/script/my-script.sh"
             env:
               - name: NAME
                 valueFrom:
                   fieldRef:
                     fieldPath: metadata.name
             volumeMounts:
               - name: my-script-volume
                 mountPath: /script
               - name: data
                 mountPath: /mnt/data
         volumes:
           - name: my-script-volume
             configMap:
               name: my-script
               defaultMode: 0744 
           - name: data
             persistentVolumeClaim:
               claimName: my-vol 
   EOF
   ``` 
   
   This workload should print a sequence of 20 lines to stdout and a file on the NFS volume.
   Then it should print the content of the file.
   
7. Print the logs of one of the workloads, run:

   ```shell
   kubectl logs -n $NAMESPACE `kubectl get pod -n $NAMESPACE -l app=gcpnfsvolume-demo -o=jsonpath='{.items[0].metadata.name}'`
   ```
   

>    The command should print something like:

Reviewer: ```suggestion
   The command should print a result similar to the following one:
```

   ```
   ...
    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 19
    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 20
    File content:
    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 1
    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 1
    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 2
    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 2
    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 3
    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 3
   ...
   ```
   

>    Note that the content after `File content:` contains prints from both workloads. This 

Reviewer: ```suggestion
   >[!NOTE] The `File content:` contains prints from both workloads. This demonstrates the ReadWriteMany capability of the volume.
```

>    demonstrates the ReadWriteMany capability of the volume.

Reviewer: ```suggestion
```
